src/test/java/org/llorllale/cactoos/matchers/MatcherOfTest.java

```diff
@@ -55,4 +64,28 @@ public void mismatchesFunc() {
             new IsNot<>(new Matches<>(1))
         ).affirm();
     }
+
+    @Test
+    public void describesMismatch() {
+        this.exception.expect(AssertionError.class);
+        this.exception.expectMessage(
+            String.format(
```

Comment:
Sometimes you use `String.format`, sometimes `new FormattedText(`, you'd better decide which style you prefer and stick to it across the commit at least. The `OOP/new` is preferred over static in those projects though.

Reply:
Changed it to `FormattedText`.